saleor/dashboard/sites/forms.py

```diff
@@ -8,18 +9,37 @@ class SiteForm(forms.ModelForm):
     class Meta:
         model = Site
         exclude = []
+        labels = {
+            'domain': pgettext_lazy(
+                'Domain name', 'Domain name'),
```

Comment:
Context should mention this being a fully qualified internet domain so it's clear what "domain" means here.